vendor/github.com/elastic/gosigar/sigar_windows.go

```diff
@@ -348,6 +348,10 @@ func (self *ProcTime) Get(pid int) error {
 	return nil
 }
 
+func (self *ProcIO) Get(pid int) error {
+	return ErrNotImplemented{runtime.GOOS}
```

Comment:
Ideally we would test freebsd to see if it has the `/proc/<pid>/io` file. If it does then no action is required because it will just work since the implementation is in `sigar_linux_common.go` which is shared with FreeBSD.

If it doesn't have a `/proc/<pid>/io` file then the implementation in  `sigar_linux_common.go` needs to move to `sigar_linux.go` and a stub that returns `ErrNotImplemented` needs added to `sigar_freebsd.go`.